### DISCUSSION TITLE
study and practice plan recommended questions

### DISCUSSION THREAD

**Discussion by BrianRuizy** [yangshun/tech-interview-handbook#271] [Q&A] [answered]
I'm confused by the new _Study and Practice Plan_ format. 

The **recommended questions** from weeks 1-4 are also found in the later weeks.
So should we not solve those recommended problems in those initial 4 weeks, and just read through the concepts?

<img width="1009" alt="Screen Shot 2022-04-20 at 9 24 06 PM" src="https://user-images.githubusercontent.com/23439187/164359089-b95c7846-5651-4974-97d2-91aacb37e98a.png">

<img width="1040" alt="Screen Shot 2022-04-20 at 9 20 47 PM" src="https://user-images.githubusercontent.com/23439187/164358833-a824fb10-4662-4377-8cb5-b7ff6488fd88.png">

**Comment by yangshun** [comment, marked as answer]
Hi @BrianRuizy, the recommended questions is a full list of questions and are for those who don't plan to follow the study plan. I think I should split up the list in the Algos page into "Do while studying" vs "Do while practicing later on" or something along those lines. Will mull over it. Thanks for the feedback!

**Reply by yangshun** [reply]
@BrianRuizy we've split up the list into "Essential questions" vs "Recommended practice questions". While studying, you just have to do the "Essential questions". The "Recommended practice questions" can be done on your own or as part of the Week 5-12 plan/Grind 75. Hope that clarifies!


Example for [Arrays](https://www.techinterviewhandbook.org/algorithms/array/#essential-questions)

<img width="870" alt="image" src="https://user-images.githubusercontent.com/1315101/168453048-c5c78a9b-d579-4f4c-aae5-77770ef301e4.png">

**Reply by BrianRuizy** [reply]
Nice work! Thank you for the follow up. @yangshun
